dipy/viz/horizon/tab/cluster.py

```diff
@@ -150,3 +149,7 @@ def build(self, tab_id, tab_ui):
     @property
     def name(self):
         return self.__name
+
+    @property
+    def tab_type(self):
+        return 'clusters_tab'
```

Comment:
To remove, you do not need it.  you can use `isinstance(cls, clstype)`  or if you love string, you can use:
- `myinstance.__class__.__name__`.
- `type(myinstance).__name__`.

Reply:
I tried using is instance but the problem is I cannot import SlicesTab class to verify. As it was creating some circular dependencies

Reply:
Can you help me understanding that and change it to isinstance. I myself is not liking the string part